app/forms/AssistanceDetailsForm.scala

```diff
@@ -26,28 +26,37 @@ object AssistanceDetailsForm {
   val isFastStream = (requestParams: Map[String, String]) =>
     requestParams.getOrElse("applicationRoute", Faststream.toString) == Faststream.toString
 
+  val isEdip = (requestParams: Map[String, String]) =>
+    requestParams.getOrElse("applicationRoute", Edip.toString) == Edip.toString
```

Comment:
With the current logic seems that doesn't really matter with method we call isEdip/isFastStream in the case the key (applicationRoute) is not in the map. They both return true. Is that the kind of behavior we want?